Apps/W1/File - Azure Blob Service Connector/app/permissions/FileStorageAdminBlobStor.PermissionSetExt.al

```diff
@@ -0,0 +1,11 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.ExternalFileStorage;
+
+permissionsetextension 80100 "File Storage - Admin - Blob Stor." extends "File Storage - Admin"
```

Comment:
Darn, we should have named the permission set External File Storage. File Storage suggests that you through this permission set can control the access to all files.

Reply:
This will have to get fixed in BCApps, if we want to change this. Or we just close both eyes and live with it 😉